andela/travel_tool_back
# #160371133 Send notification after creating a request

#### What does this PR do?
Adds a method that saves notifications to the db and emits notifications to the client

#### Description of Task to be completed?
- add `sendNotitifcationToMnager` method that is responsible for sending a notification

#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_back.git`
- Enter into the project directory - `cd travel_tool_back`
- Install dependencies and start the server - `make start`
- make yourself a manager
- create a new request with yourself as the manager
- you should see the notification on your notification pane

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#160371133](https://www.pivotaltracker.com/story/show/160371133)

#### Screenshots (if appropriate)
None

#### Questions:
None
